src/server/analytic_services/endpoint_stats_store.js

```diff
@@ -11,7 +11,7 @@ const endpoint_group_report_schema = require('./endpoint_group_report_schema.js'
 const { ENDPOINT_MONITOR_INTERVAL } = require('../../../config');
 
 class EndpointStatsStore {
-    static get instance() {
+    static instance() {
```

Comment:
Why? I don't see the reasoning behind this change.
instance as a property makes more sense, and functionally they are the same.

Reply:
Rolledback